packages/buttons/src/views/Anchor.js

```diff
@@ -60,9 +59,9 @@ export const StyledNewWindowIcon = styled(NewWindowIcon)`
 /**
  * Accepts all `<a>` props
  */
-const Anchor = props => {
-  // eslint-disable-next-line
-  const { focused, external, children, theme, ...other } = props;
+const Anchor = React.forwardRef((props, ref) => {
+  // eslint-disable-next-line no-unused-vars
+  const { focused, external, children, theme, ...other } = props; // eslint-disable-line react/prop-types
```

Comment:
Why is theme destructured here?

Reply:
Great question. Not sure why it was like this. I've removed it.